src/blog/three-new-features-on-ganache-to-improve-your-developer-experience/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Three new features on Ganache to improve your developer experience
+hide:
+  - navigation
+---
+
+**By [Kingsley Arinze](https://twitter.com/heydamali)**
+
+![The Merge and what it means for Truffle](./what-is-new-in-ganache.jpg)
+
+Since the release of Ganache 7, we’ve mostly focused our attention on bug fixes and UX improvements. Now with the changes coming to Ethereum in the coming weeks, due to the Merge, we’ve decided to expand our focus to also include new features that not only support the Merge and layer two networks, but set the foundation for subsequent upgrades for Truffle post-merge.
+
+In addition, with the migration of Ethereum from PoW to PoS and the separation of the consensus layer from the execution layer, our users have been curious to know how Ganache is impacted and what actions they will need to take. 
+
+The simple answer to this very important question is: although [there are changes to how Ethereum clients will work post-merge](https://consensys.net/blog/ethereum-2-0/the-four-pillars-of-the-merge-to-proof-of-stake-how-ethereum-will-evolve), Ganache users do not need to do anything since these changes to Ethereum will not affect the user and dapp development experience.
+
+In this blog, we’ll touch on 3 new features that have been added to Ganache since the Ganache 7 release in January. To be able to use some of these features, you will need to run the latest version of Ganache. You can download it from [here](https://trufflesuite.com/ganache).
+
+## Zero config mainnet forking now available in the browser
+
+For some time now, it has been possible to run Ganache on the browser by simply including a minimal  javascript code in your HTML and then accessing the Ganache provider as shown below:
+
+```javascript
+<script src="https://cdn.jsdelivr.net/npm/ganache@{VERSION}/dist/web/ganache.min.js"></script>
+
+const options = {};
+const provider = Ganache.provider(options);
+```
+Although adding the above lines of code would make the Ganache blockchain simulator available in your browser, features like the zero configuration mainnet forking wasn’t available.
+
+However, with the latest release, Ganache users can now enjoy this feature in their browser by simply specifying additional options during setup as follows:
```

Comment:
```suggestion
Since the release of Ganache v7.3.2 users can now utilize this feature in the browser by specifying additional options during setup:
```